src/status_im2/contexts/chat/composer/sub_view.cljs

```diff
@@ -48,10 +48,12 @@
          :label               (i18n/label :t/jump-to)
          :style               {:align-self :center}}}
        {}]]
-     [quo/floating-shell-button
-      (when @messages.list/show-floating-scroll-down-button
-        {:scroll-to-bottom {:on-press messages.list/scroll-to-bottom}})
-      {}]]))
+     (when @messages.list/show-floating-scroll-down-button
+       [quo/floating-shell-button
+        {:scroll-to-bottom {:on-press messages.list/scroll-to-bottom}}
+        {:position :absolute
```

Comment:
tiny nit pick : styles could be moved over to `status-im2.contexts.chat.composer.style`

Reply:
done, thank you